articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Feature Specific Setup with Custom Domains
+description: How to set up custom domains with feature specific instructions
+toc: true
+---
+# Feature-Specific Setup
+
+There are additional steps you must complete depending on which Auth0 features you are using.
```

Comment:
This intro needs work. I understand that this used to be part of the custom-domains main doc, and it might have made sense like that there, but here it makes absolutely no sense. You should describe what this doc is about. Personally, judging from the title and this first sentence, I have no idea what to expect.